src/lib/datocms/tests/datocmsCollection.test.ts

```diff
@@ -29,36 +30,60 @@ afterEach(() => {
 afterAll(() => server.close());
 
 describe('datocmsCollection:', () => {
-  test('should successfully fetch a non-paginated collection', async () => {
+  const fragment = parse(/* graphql */`
+    fragment MyMockRecordFragment on MyMockRecord {
+      id
+      title
+    }
+  `);
+
+  test('supports passing fragment as a string', async () => {
     const totalRecords = 50;
     const mockCollection = Array.from({ length: totalRecords }, (_, i) => ({
       id: i + 1,
       title: `Record ${i + 1}`
     }));
 
-    server.use(
-      graphql.query('MyMockCollectionMeta', () => {
-        return HttpResponse.json({
-          data: {
-            meta: {
-              count: totalRecords,
-            },
-          },
-        });
-      }),
-    );
+    server.use(graphql.query('MyMockCollectionMeta', () => HttpResponse.json({
+      data: {
+        meta: { count: totalRecords },
+        records: [{ __typename: 'MyMockRecord' }],
+      } satisfies CollectionInfo,        
```

Comment:
Make sure the payload for the mocked service is the same type as the expected return value